x-pack/test/functional/page_objects/remote_clusters_page.ts

```diff
@@ -29,6 +29,12 @@ export function RemoteClustersPageProvider({ getService }: FtrProviderContext) {
       });
       await testSubjects.setValue('remoteClusterFormNameInput', name);
       await comboBox.setCustom('comboBoxInput', seedNode);
+
+      // Complete trust setup
```

Comment:
Good catch! I've moved the calls around to accomodate that, the skipped tests are part of another issue https://github.com/elastic/kibana/issues/153599 but I think is a bit out of scope for this PR so I'll leave it skipped and we can pick it up after.

Reply:
Yes, the test was skipped because of an accessibility issue, but the `createNewRemoteCluster()` function which is part of the test setup was working correctly before these changes - it fails now because it cannot navigate the "Yes, I have setup trust" checkbox.

I was thinking that we could leave the test skipped but make sure that the `createNewRemoteCluster()` function executes successfully as before. Wdyt?

Reply:
Yes! I've patched it up but haven't pushed changes yet as I'm addressing other feedback I got for this pr!

Reply:
Got it, thanks!